status-im/status-react
# Show accurate Swap approval fee

fixes #21707

### Summary

follow-up to #21541

The Swap approval fee we show wasn't consistent with the fee from desktop. In this PR we show the fee we get from the router instead of calculating it on our own.

### Steps to test


- Open Status
- Log in and open wallet
- Start Swap with Optimism with an ERC20 token
- Choose ETH as receiver
- Observe fee (compare with desktop)

status: ready
